db/migrate/20240309152840_create_user_settings.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUserSettings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :user_settings do |t|
+      t.string :preferred_languages
```

Comment:
Shouldn't this be `preferred_language` ? (Singular)

Reply:
Yes, I decided to fix this directly in the migration file as it is the first PR of the project and we can still just drop/re-create the db; I'm aware that later we would need to write a migration for this 🫡

Reply:
When is a PR it doesn't matter. Since we haven't pulled your code and mergeit in our local DB, so you can go nuts will all migrations you write in your PR 🙌